lonestar/mining/cpu/subgraph-listing/README.mb

```diff
@@ -0,0 +1,37 @@
+Subgraph Listing
+================================================================================
+
+DESCRIPTION 
+--------------------------------------------------------------------------------
+
+This application counts the occurances of a given subgraph in a graph. 
+Currently only two patterns are supported: diamond and 4-cycle.
+
+INPUT
+--------------------------------------------------------------------------------
+
+This application takes in symmetric and simple Galois .gr graphs.
+You must specify both the -symmetricGraph and the -simpleGraph flags when
+running this benchmark.
+You must also specify the query graph (i.e. pattern) using -p.
```

Comment:
Please remove this line.

Reply:
this is required

Reply:
Isn't the query graph a cycle or a diamond depending on the executable? I don't understand what needs to be specified. Can you please add an example?

Reply:
Examples are already there. The commandline format is designed for any arbitrary pattern. The current implementation is ad-hoc so that the executables are limited to a specific pattern. All executables will be merged into one soon, in which case -p is used to specify the pattern.

Reply:
> The current implementation is ad-hoc so that the executables are limited to a specific pattern

That is my point. When you merge into one executable, you can reintroduce that line in the README. For now, the README and the executables are not consistent with each other.

Reply:
which part is not consistent?